### PR TITLE
Changes to successfully package and upload beta app to the store.  

### PR DESCRIPTION
**Pull request by nishitha-burman**
Changes in appxmanifest and logos for the beta app.

### CODE REVIEW

File: version.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "7.0.0-build.{height}",
+  "version": "7.0.0-preview4",
```
Comment (simeoncran) [inline version.json:2]: This file needs to be reverted. This change is just a normal change, it's not a preview release. This change would turn it into a new preview4.

File: LottieViewer/LottieViewer.csproj [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
     <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.18362.0</TargetPlatformVersion>
+    <AppxPackageSigningTimestampDigestAlgorithm>SHA256</AppxPackageSigningTimestampDigestAlgorithm>
+    <GenerateTestArtifacts>True</GenerateTestArtifacts>
+    <HoursBetweenUpdateChecks>0</HoursBetweenUpdateChecks>
   </PropertyGroup>
-
   <PropertyGroup>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <AppInstallerCheckForUpdateFrequency>OnApplicationRun</AppInstallerCheckForUpdateFrequency>
-    <AppInstallerUpdateFrequency>1</AppInstallerUpdateFrequency>
```
Comment (simeoncran) [inline LottieViewer/LottieViewer.csproj:15]: All of the changes up to line 130 should be reverted to what they were before.

**Review by simeoncran — CHANGES_REQUESTED** [review]
:clock1:

File: LottieViewer/LottieViewer.csproj (hunk 2 above)
Comment (nishitha-burman) [reply LottieViewer/LottieViewer.csproj:15]: Do you mean line 13?

**Review by simeoncran — APPROVED** [review]
:shipit:

**Pull request merged**